app/assets/javascripts/discourse/app/pre-initializers/inject-discourse-objects.js

```diff
@@ -99,10 +87,11 @@ export default {
       app.inject(t, "session", "session:main");
       app.inject(t, "messageBus", "message-bus:main");
       app.inject(t, "siteSettings", "site-settings:main");
+      app.inject(t, "topicTrackingState", "topic-tracking-state:main");
     });
 
     if (currentUser) {
-      ["component", "route", "controller", "service"].forEach((t) => {
+      ["controller", "component", "route", "service"].forEach((t) => {
```

Comment:
just making them alphabetical?

Reply:
Made the order match the `ALL_TARGETS` array at the top of the file. Had a hard time comparing them visually. 😂